server/db/recreate-schema.sql

```diff
@@ -27,14 +27,24 @@ CREATE TABLE languages (
 );
 
 CREATE TABLE documents (
-  id               SERIAL PRIMARY KEY,
-  name               VARCHAR(60) NOT NULL,
+  id                   SERIAL PRIMARY KEY,
+  name                 VARCHAR(60) NOT NULL,
   from_language_code   VARCHAR(6) REFERENCES languages(code) NOT NULL,
   to_language_code     VARCHAR(6) REFERENCES languages(code) NOT NULL,
-  status           document_status,
-  submission_date  DATE NOT NULL,
-  due_date         DATE NOT NULL,
-  owner_id         uuid REFERENCES users(id) NOT NULL,
-  format           format_types,
-  content          TEXT NOT NULL
+  status               document_status,
+  submission_date      DATE NOT NULL,
+  due_date             DATE NOT NULL,
+  owner_id             uuid REFERENCES users(id) NOT NULL,
+  format               format_types,
+  content              TEXT NOT NULL
 );
+
+CREATE TABLE translations (
```

Comment:
This table needs also to be deleted at the top of the file to avoid problem when recreating the DB.

Reply:
Done